circle/functions

```diff
@@ -457,43 +467,67 @@ add_repo_to_helm() {
   helm repo add $REPO_NAME $REPO_URL
 }
 
-chart_update_image() {
-  local CHART_NEW_IMAGE_VERSION=${2#*:}
-  local CHART_CURRENT_IMAGE_VERSION=$(grep "^[ ]*image:[ ]*${2%:*}" ${1}/values.yaml)
-  local CHART_CURRENT_IMAGE_VERSION=${CHART_CURRENT_IMAGE_VERSION##*:}
+chart_update_image_tag() {
+  local CHART_PATH=${1}
+  local CHART_NEW_IMAGE_TAG=${2}
+  local CHART_IMAGE_REPOSITORY=${3}
+  local CHART_CURRENT_IMAGE_TAG="null"
 
-  case $(vercmp $CHART_CURRENT_IMAGE_VERSION $CHART_NEW_IMAGE_VERSION) in
+  // As Tensorflow and Parse chart uses two docker images we need to update the proper image tag
+  case ${CHART_IMAGE_REPOSITORY} in
+    "parse" | "tensorflow-serving" )
+      CHART_CURRENT_IMAGE_TAG=$(yq -r .server.image.tag ${CHART_PATH}/values.yaml)
```

Comment:
Note that if we use the `select` function, we can return an empty string if the key does not exist. f.e. `yq -r 'select(.server.image.tag != null) | .server.image.tag' ${CHART_PATH}/values.yaml`.